src/datasets/other/stub_installer/reference.md

```diff
@@ -0,0 +1,26 @@
+# What is the Stub Installer ping?
+
+When the stub installer completes with almost any result, it generates a ping containing some data about the system and about how the installation went. This ping isn't part of Firefox unified telemetry, it's a bespoke system; we can't use the telemetry client code when it isn't installed yet.
```

Comment:
On what results does it not send a ping?

Reply:
According to [this](https://searchfox.org/mozilla-central/source/browser/installer/windows/docs/StubPing.rst), "No ping is sent if the installer exits early because initial system requirements checks fail." I can add that line to DTMO.

Reply:
👍